packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -1577,15 +1577,26 @@ export async function saveFile(doc: vscode.TextDocument, datasetProvider: api.IZ
     const ending = doc.fileName.substring(start);
     const sesName = ending.substring(0, ending.indexOf(path.sep));
     const profile = Profiles.getInstance().loadNamedProfile(sesName);
+    const fileLabel = doc.fileName.split("/").slice(-1)[0];
```

Comment:
This line should also be split by `path.sep` instead of a single forward slash - on Windows, this split operation may fail to return the right path segments.

Reply:
Was this something introduced with this PR or existing and planning to address for `v2.12.2`? @traeok

Reply:
I just found this issue during my review, but it's not a part of this PR's changes. We'll want this in the v2.12.2 release - I just realized this line is only used for the file label, so it shouldn't be a huge regression, but the editor tabs might look odd on Windows for v2.12.1.

We can merge a fix into maintenance, and then update this PR with the changes from the base branch.

Reply:
I think we can bring in the changes from a possible `2.12.2` if we do so as a separate PR since we may want to bring these changes into our `main` branch if that's ok with you?

Reply:
Addressed this in https://github.com/zowe/vscode-extension-for-zowe/commit/eed67ec8c9945038a75f8b54a015fa995bddaac6